README.md

```diff
@@ -1,6 +1,8 @@
 <!--![shiba love MRI scanner](https://raw.githubusercontent.com/learn-neuroimaging/tutorials-and-resources/master/img/dogeLoveScanner_600x400.gif)-->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-21-orange.svg?style=flat-square)](#contributors-)
+![Github All Contributors](https://img.shields.io/github/all-contributors/learn-neuroimaging/tutorials-and-resources/master?color=orange&style=flat-square) [![Build Status](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources.svg?branch=master)](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources)
```

Comment:
```suggestion
![Github All Contributors](https://img.shields.io/github/all-contributors/learn-neuroimaging/tutorials-and-resources/master?color=blue) [![Build Status](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources.svg?branch=master)](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources)
```

Since we will now have new badges, we should keep the style across badges (I like rounded corners personally 😇 ). I let you chose the final style as your prefer.

Once done, I will merge your PR :)

Reply:
Great...I prefer rounded too :innocent: